src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -735,4 +735,13 @@ Parameter name: {0}</value>
     <value>The current user, '{0}', is not a maintainer of the package '{1}'.</value>
     <comment>{0} = user name (i.e. 'DynamoTeam'), {1} = package name (i.e. 'Clockwork for Dynamo 1.x')</comment>
   </data>
+  <data name="DuplicatedOlderPackage" xml:space="preserve">
+    <value>An older version of the package called {0} version {2} was found at {1} with version {3}. The older version has been ignored.</value>
+  </data>
+  <data name="InvalidPackageVersion" xml:space="preserve">
+    <value>The version of the package called {0} found at {1} is invalid (version: "{2}"). Ignoring it.</value>
+  </data>
+  <data name="DuplicatedPackageAlreadyLoaded" xml:space="preserve">
+    <value>A newer version of the package called {0} version {2} was found at {1} with version {3}. The old version of the package is already loaded. Please restart Dynamo for loading the new version of the package.</value>
```

Comment:
is the application always "Dynamo" ? ..in ` Please restart Dynamo for`

Reply:
I think so but it is a good point.

Reply:
yeah, what if we're in Revit - you actually need to reboot Revit...

Reply:
Ok, I was following the messaging that we have for package manager that currently says "Restart Dynamo to complete the uninstall.". I can change to "Please restart for loading"?